aten/src/ATen/cpu/vec/vec256/functional.h

```diff
@@ -231,4 +231,444 @@ inline void map3(
   }
 }
 
+// BFloat16 specification
```

Comment:
Please move bfloat16 related code into the `vec256_bfloat16.h`

Reply:
If necessary (circular deps) create another file, something like _mixed.h

Reply:
Done. Separate bfloat16 part into `functional_bfloat16.h`